practice-one/src/assets/css/footer.css

```diff
@@ -35,26 +30,31 @@
   flex-direction: row;
   gap: 54px;
 }
+
 .follow-us {
   display: flex;
   gap: 43px;
 }
+
 .break {
   max-width: 133px;
   max-height: 77px;
 }
+
 .break-about-us {
   max-width: 360px;
   max-height: 48px;
 }
-.rule {
+
+.footer-rule {
```

Comment:
Why do we need this class? Do we have another way to make the line in the footer?